lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -61,16 +61,14 @@
   t.pattern = 'spec/acceptance'
   # TEST_TIERS env variable is a comma separated list of tiers to run. e.g. low, medium, high
   if ENV['TEST_TIERS']
-    tiers = '--tag '
-    test_tiers = ENV['TEST_TIERS'].split(',')
+    test_tiers = test_tiers.split(',')
```

Comment:
This is splitting on itself.  Shouldn't this still be `test_tiers = ENV['TEST_TIERS'].split(',')` ?

Reply:
Yup, I removed a gsub call from earlier that botched this up. Thanks, Glenn